kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -873,6 +873,14 @@ public Properties getProducerProperties() {
 
     Properties producerProperties = new Properties();
     producerProperties.putAll(producerConfigs);
+
+    // KREST-4606: Disable idempotency until KAFKA-13668 is fixed.
+    if (Boolean.parseBoolean(producerProperties.getProperty("enable.idempotence"))) {
+      log.warn(
+          "Idempotent producers are not supported in Kafka REST. Ignoring enable.idempotence'.");
```

Comment:
nit: A single quote seems to be missing just before _enable_.

Reply:
I'm not quite decided, but is warn strong enough here?  If someone thinks they are getting idempotence, and they are not, that's quite serious isn't it?  If it's something that requires once only delivery of messages isn't turned on, even after they set the setting is set to on (eg payment transfer happens twice instead of once) that could have consequences.

What do you think?

Reply:
There are already no guarantees, due to HTTP retries, so I don't think it is very serious. `error` should only be used for things that are critical. In this case, if we did `error`, we should all exit the application. Since we aren't, I think `warn` is good enough.